src/instances.ts

```diff
@@ -391,11 +407,27 @@ export function initializeInstance(
         instance.languageService!.getProgram()
       );
     }
+    if (loader._compilation.hooks.afterProcessAssets) {
+      // afterProcessAssets does not exist in webpack4
+      loader._compilation.hooks.afterProcessAssets.tap(
+        'ts-loader',
+        (_: any) => {
```

Comment:
Do you happen to know what the type is here? Not that we're using it but I'm curious

Reply:
It is a list of the assets:

https://github.com/webpack/webpack/blob/9342540a1f11b84cd693b186ec4e72d37f5a0aeb/lib/Compilation.js#L299

If I include the parameter there is a warning that it is unused, so I decided to include it as <code>_:any</code>